include/sdf/Types.hh

```diff
@@ -35,7 +36,12 @@ namespace sdf
   inline namespace SDF_VERSION_NAMESPACE {
   //
 
-  const std::string kSdfScopeDelimiter = "::";
+  static const std::string &SdfScopeDelimiter()
```

Comment:
Per the [GSG](https://google.github.io/styleguide/cppguide.html#Internal_Linkage), we should avoid using `static` for the function here as well as for `kSdfScopeDelimiter` since this is a header file.

Reply:
Thanks for helping with this! https://github.com/gazebosim/sdformat/pull/1265